tests/unit/test-libm.c

```diff
@@ -39,7 +39,14 @@ check_double (const char *expr, double computed, double expected)
   uint64_t *computed_bits_p = (uint64_t *) &computed;
   uint64_t *expected_bits_p = (uint64_t *) &expected;
 
-  printf ("%s = 0x%lx [expected=0x%lx] ", expr, *computed_bits_p, *expected_bits_p);
+  uint32_t computed_hi = (uint32_t) (*computed_bits_p >> 32);
+  uint32_t computed_lo = (uint32_t) *computed_bits_p;
+  uint32_t expected_hi = (uint32_t) (*expected_bits_p >> 32);
+  uint32_t expected_lo = (uint32_t) *expected_bits_p;
```

Comment:
I've rewritten it as suggested. See updated patch. (Note, however, that it's not as big/little-endian-safe. But it's just the printing, so no big deal.)